resources/views/organisations/datagrid.blade.php

```diff
@@ -17,10 +17,13 @@
         ],
         // Location
         [
-            'type' => 'avatar',
-            'parent' => 'location',
-            'parent_route' => 'locations',
-            'visible' => $campaignService->enabled('locations'),
+            'label' => __('quests.fields.quest'),
```

Comment:
I'm not sure why organisations were changed? Organisations don't have a parent quest

Reply:
It was a mistake, its been reverted.